### PR TITLE
Update GKE.md

### PR DESCRIPTION
**Pull request by Nino-K**
Changes project name to project id as get-credentials command requires project-id.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: tests/integration/GKE.md [hunk 1]
```diff
@@ -77,7 +78,7 @@ gcloud container clusters \
 
 ```bash
 gcloud container clusters get-credentials ${CLUSTER_NAME} \
-   --zone ${ZONE} --project ${PROJECT_NAME}
+   --zone ${ZONE} --project ${PROJECT_ID}
```
Comment (nmittler) [inline tests/integration/GKE.md]: The documentation in gcloud says that it's the project name, not the ID.

File: tests/integration/GKE.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ gcloud container clusters \
  - `CLUSTER_NAME`: Whatever suits your fancy, 'istio-e2e' is a good choice.
  - `ZONE`: 'us-central1-f' is a good value to use.
  - `PROJECT_NAME`: is the name of the GCP project that will house the cluster. You get a project by visiting [GCP](https://console.cloud.google.com).
+ - `PROJECT_ID`: is the ID of the GCP project that will house the cluster. You get a project by visiting [GCP](https://console.cloud.google.com).
```
Comment (nmittler) [inline tests/integration/GKE.md]: Are you replacing PROJECT_NAME? If so, delete the old value.

Also, you need to update the description above.

File: tests/integration/GKE.md (hunk 1 above)
Comment (Nino-K) [reply tests/integration/GKE.md]: It doesn't work with project name, it only works with project_ID

Comment (nmittler) [reply tests/integration/GKE.md]: Interesting ... I was sure I used name in the past.  Whatever ... just delete the name and update example above.

File: tests/integration/GKE.md (hunk 2 above)
Comment (Nino-K) [reply tests/integration/GKE.md]: @nmittler The description says ID instead of name, is there any other changes you had in mind that you want me to add?

**Pull request merged**